src/reducers/platform.js

```diff
@@ -24,21 +24,23 @@ export function app(state=DEFAULT_APP_STATE, action={}) {
 
 	switch (action.type) {
 	case 'API_REQUEST':
-		if (action.meta.logout) {
+		if ((action.meta || {}).logout) {
```

Comment:
this is sort of an unrelated change, but prevents an annoying requirement for `apiRequest` calls to always supply a `meta` argument.